metabase/metabase
# fix(sdk): entity picker theming fixes

Closes https://github.com/metabase/metabase/issues/50394

### Description

Adds additional theming fixes for the entity picker so it follows the SDK theming.

### How to verify

- Open Storybook on the SDK and click on the "Dark Theme Editor" stories

![CleanShot 2567-11-25 at 18 45 08@2x](https://github.com/user-attachments/assets/f309d9d9-f4ba-4d6b-885b-ab272f5cf5c8)

- There should be no noticeable change in the non-themed "Editor Only" story. Remember to _reload_ the page after switching between stories, as there is a bug caused by patching the `colors` object that caused the themed colors to persist across stories.

### Demo

Themed Chevron Color

<img src="https://github.com/user-attachments/assets/a55dceab-5f36-4fcf-8ef1-954f6311e009" width="300">

Themed Segment Control

![CleanShot 2567-11-25 at 18 40 32@2x](https://github.com/user-attachments/assets/91adfa48-a099-4556-bb85-4fe775556234)

Themed Skeleton: [see private Loom](https://www.loom.com/share/ac01f71df753402abc4592577294f8f1?sid=05d4101c-7a17-4071-9b8c-3094d7a81394)
